media/css/firefox/challenge-the-default/_animations.scss

```diff
@@ -0,0 +1,121 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$campaign-red: #ff6a75;
+$campaign-pink: #ffbdc5;
+$campaign-mint: #d1ffee;
+$campaign-green: #2ac3a2;
+
+.js {
+    @media (prefers-reduced-motion: no-preference) {
+
+        @keyframes pop-in {
+            0% {
+                opacity: 0;
+                transform: scale(0.3);
+            }
+
+            35% {
+                opacity: 1;
+                transform: scale(0.3);
+            }
+
+            50% {
+                transform: scale(0.7);
+            }
+
+            65% {
+                transform: scale(0.7);
+            }
+
+            80% {
+                transform: scale(1.1);
+            }
+
+            90% {
+                transform: scale(1.1);
+            }
+
+            100% {
+                opacity: 1;
+                transform: scale(1);
+            }
+        }
+
+        @keyframes close {
+            0% {
+                opacity: 1;
+                transform: scale(1);
+            }
+
+            50% {
+                opacity: 0.5;
+                  transform: scale(0.7);
+            }
+
+            100% {
+                opacity: 0;
+                 transform: scale(0.3);
+            }
+        }
+
+        @keyframes wednesday-circles-mint {
+            0% {
+                fill: $campaign-green;
+            }
+
+            25%, 100% {
+                fill: $campaign-mint;
+            }
+        }
+
+        @keyframes wednesday-circles-red {
+            0% {
+                fill: $campaign-green;
+            }
+
+            25%, 100% {
+                fill: $campaign-red;
+            }
+        }
+        @keyframes wednesday-circles-pink {
+            0% {
+                fill: $campaign-green;
+            }
+
+            25%, 100% {
+                fill: $campaign-pink;
+            }
+        }
+
+
+
+
+        .animate-close {
+            animation: .5s ease-out 0s 1 normal forwards running close;
```

Comment:
**Suggestion:** I have a strong inclination that the `close` animation should be faster, which helps with the smoothness of my (suggested above) animation scales. I tested it out with `0.3s` and it creates a much better experience:

```suggestion
            animation: .3s ease-out 0s 1 normal forwards running close;
```

Reply:
Thanks!!